src/qt/forms/helpmessagedialog.ui

```diff
@@ -6,33 +6,15 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>585</width>
-    <height>225</height>
+    <width>780</width>
```

Comment:
Thanks, I already asked if the resize in the initial pull was done by intention and got a yes. It was too small IMHO, so good one here!